### PR TITLE
Re bet&&mert

### PR DESCRIPTION
**Pull request by motuer97**
Some updates

### CODE REVIEW

File: data/hlm/dataloader.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    def loadTrainData(self, **kwargs):
+        pass
+
+    def loadTestData(self, **kwargs):
+        pass
+
+
+if __name__ == "__main__":
+    pass
```
Comment (9DemonFox) [inline data/hlm/dataloader.py:19]: why you define a empty class

**Comment by 9DemonFox** [comment]
have you add any doc

**Review by 9DemonFox — APPROVED** [review]

**Comment by motuer97** [comment]
hlm是许中飞在写的的模型 ，本来就是空的好像
，我只是单纯merge到我本地来了

---Original---

**Comment by 9DemonFox** [comment]
You may make mistakes in some operations，you need only to merge from master。And the green area indicate it is added by you

**Pull request merged**